stanford_media.install

```diff
@@ -57,3 +57,93 @@ function stanford_media_update_8003() {
   $display->setComponent('field_media_video_embed_field', $display_options);
   $display->save();
 }
+
+/**
+ * Move inline image alt text into a contrib supported attribute.
+ */
+function stanford_media_update_8004() {
+  foreach (_stanford_media_update_8004_get_filter_fields() as $field) {
+    list($entity_type, $field_name) = explode(':', $field);
+
+    try {
+      $entity_ids = \Drupal::entityQuery($entity_type)
+        ->condition($field_name, '%alt_text&quot%', 'LIKE')
+        ->execute();
+    }
+    catch (Exception $e) {
+      continue;
+    }
+
+    $entity_storage = \Drupal::entityTypeManager()->getStorage($entity_type);
+    foreach ($entity_storage->loadMultiple($entity_ids) as $entity) {
+      $field_values = $entity->get($field_name)->getValue();
+
+      foreach ($field_values as &$field_value) {
+        $field_value['value'] = _stanford_media_update_8004_fix_alt($field_value['value']);
+      }
+
+      $entity->set($field_name, $field_values);
+      $entity->save();
```

Comment:
This is not taking into account revisions, is it?

Reply:
No. i'm less worried about revisions. worst case is they revert to an older revision and an image won't have an alt text. but 99.9% of our clients never do anything with revisions

Reply:
Aye, but some other modules that handle workflow related stuff use revisions to handle drafts and such. It is possible that they could be put in limbo but from what we know about this module's usage that is unlikely. Leave a comment in the update hook that calls this out so it is visible when running.